### ISSUE TITLE
Allow escaping of ampersand in attribute selectors

### ISSUE THREAD

**Issue by NeonPaul** [postcss/postcss-nested#55]
I want to be able to write a rule like this:

```css
.my-room {
  &[data-category='Sound & Vision'] {
    color: $electric-blue;
  }
}
```

Currently  the second `&` is replaced with the current selector.

Please either add the option to escape an ampersand or automatically ignore them in attribute selectors.

**Comment by ai** [comment]
Good idea. Maybe we should use `postcss-selector-parser` to ignore strings. Do you want to make PR?

**Comment by NeonPaul** [comment]
OK this looks quite straight forward. I will try and make a PR when I'm not busy.

**Comment by ivantsov** [comment]
Looks like this change caused a bug - https://github.com/postcss/postcss-nested/issues/58.

**Comment by ai** [comment]
Fixed in 2.0.3
